website/source/guides/outage.html.markdown

```diff
@@ -0,0 +1,189 @@
+
+If [`nomad server-force-leave`](/docs/commands/server-force-leave.html) isn't
+able to remove the server, you have two methods available to remove it,
+depending on your version of Nomad:
+
+* In Nomad 0.5.5 and later, you can use the [`nomad operator raft
+  remove-peer`](/docs/commands/operator-raft-remove-peer.html) command to remove
+  the stale peer server on the fly with no downtime.
+
+* In versions of Nomad prior to 0.5.5, you can manually remove the stale peer
+server using the `raft/peers.json` recovery file on all remaining servers. See
+the [section below](#peers.json) for details on this procedure. This process
+requires Nomad downtime to complete.
+
+In Nomad 0.5.5 and later, you can use the [`nomad operator raft
+list-peers`](/docs/commands/operator-raft-list-peers.html) command to inspect
+the Raft configuration:
+
+```
+$ nomad operator raft list-peers
+Node                   ID               Address          State     Voter
+nomad-server01.global  10.10.11.5:4647  10.10.11.5:4647  follower  true
+nomad-server02.global  10.10.11.6:4647  10.10.11.6:4647  leader    true
+nomad-server03.global  10.10.11.7:4647  10.10.11.7:4647  follower  true
+```
+
+## Failure of Multiple Servers in a Multi-Server Cluster
+
+In the event that multiple servers are lost, causing a loss of quorum and a
+complete outage, partial recovery is possible using data on the remaining
+servers in the cluster. There may be data loss in this situation because multiple
+servers were lost, so information about what's committed could be incomplete.
+The recovery process implicitly commits all outstanding Raft log entries, so
+it's also possible to commit data that was uncommitted before the failure.
+
+See the [section below](#peers.json) for details of the recovery procedure. You
+simply include just the remaining servers in the `raft/peers.json` recovery file.
+The cluster should be able to elect a leader once the remaining servers are all
+restarted with an identical `raft/peers.json` configuration.
+
+Any new servers you introduce later can be fresh with totally clean data directories
+and joined using Nomad's `server-join` command.
+
+In extreme cases, it should be possible to recover with just a single remaining
+server by starting that single server with itself as the only peer in the
+`raft/peers.json` recovery file.
+
+Prior to Nomad 0.5.5 it wasn't always possible to recover from certain
+types of outages with `raft/peers.json` because this was ingested before any Raft
+log entries were played back. In Nomad 0.5.5 and later, the `raft/peers.json`
+recovery file is final, and a snapshot is taken after it is ingested, so you are
+guaranteed to start with your recovered configuration. This does implicitly commit
+all Raft log entries, so should only be used to recover from an outage, but it
+should allow recovery from any situation where there's some cluster data available.
+
+<a name="peers.json"></a>
```

Comment:
You don't need to do this - there will be an automatic `<a>` link generated as `#manual-recovery-using-peers.json`